ui/pages/styleguide.js

```diff
@@ -0,0 +1,107 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import HeaderFooter from '../components/header-footer';
+import Link from '../components/link';
+import LabeledText from '../components/labeled-text';
+
+const GITHUB_URL = 'https://github.com/18F/omb-eregs';
+
+function Example({ children }) {
```

Comment:
I _think_ this component is meant to wrap app components so we can see clearly where they start and end. Is that right? Should it take a title in that case?

It might also be worth noting why inline styles are the way to go here.

Reply:
Oh, uh, that's mostly lazyness on my part.  Although I also figured this is the only place the styles will actually be used, so maybe it's just easier to inline them? Eh, I'm happy to factor them out into a SCSS thingy.

Reply:
Hmm, the example component is really just meant as a sort of "template partial" to show an example snippet of code--it can be just one component, or it can be a bunch of them, or it can also be things that aren't very component-like at all, such as the first example which just shows typography.  I think the equivalent in CALC did have an *optional* title, but it wasn't required...

Reply:
Re styling: I could see an argument around inlining these so they aren't overridden by one of the imported stylesheets... or something along those lines. If that's not the reason, SCSS (or basscss classes) would be better.

Re title: Gotcha. If you can think of a way to document that sentiment, it might be helpful. Having more examples may resolve the uncertainty, too, though, so we might just wait.

Reply:
Ok, I added a styleguide SCSS file. I think that ideally it would be a separate SCSS file that's only loaded at `/styleguide` so that it doesn't bloat the core CSS bundle, but that would take some more work and it's a really tiny SCSS file... maybe we could file that as a separate issue to tackle later?

Reply:
Created #743 to track, though I made it a bit more general than this one page.